imagehash/__init__.py

```diff
@@ -38,8 +38,9 @@
 #import scipy.fftpack
 #import pywt
 import os.path
-__version__ = open(os.path.join(os.path.abspath(
-	os.path.dirname(__file__)), 'VERSION')).read().strip()
+with open(os.path.join(os.path.abspath(
+	os.path.dirname(__file__)), 'VERSION')) as fi:
+	__version__ = fi.read().strip()
```

Comment:
@JohannesBuchner Note quite sure that I've followed your in-file use of tabs, let me know if I need to change anything, thanks.